dotnet-interactive/IJupyterKernelSpecInstaller.cs

```diff
@@ -8,8 +8,8 @@ namespace Microsoft.DotNet.Interactive.App
 {
     public interface IJupyterKernelSpecInstaller
     {
-        Task<KernelSpecInstallResult> InstallKernel(DirectoryInfo kernelSpecPath, DirectoryInfo destination = null);
+        Task<bool> InstallKernel(DirectoryInfo kernelSpecPath, DirectoryInfo destination = null);
 
-        Task<KernelSpecInstallResult> UninstallKernel(DirectoryInfo kernelSpecPath);
+        Task<bool> UninstallKernel(string kernelspecName);
```

Comment:
The more common naming convention would be `TryUninstallKernel`.

Reply:
Removed api

Reply:
Done